api/etcdserverpb/rpc.proto

```diff
@@ -8,10 +8,30 @@ import "etcd/api/versionpb/version.proto";
 
 // for grpc-gateway
 import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
 
 option (gogoproto.marshaler_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  security_definitions: {
+    security: {
+      key: "ApiKey";
```

Comment:
I see the key is "`ApiKeyAuth`" in the PR [547](https://github.com/grpc-ecosystem/grpc-gateway/pull/547), I am  not sure whether we should use the same name. Since you follow the same name as [main.go#L52](https://github.com/hexfusion/schwag/blob/3ceb0191ccbf98e774899d497c7120d98aa5f2b5/main.go#L52), so it should be fine and can be discussed separately.

Reply:
Yeah. I am trying to keep it as the same with original, since the diff is too much to review 😂 .